hack/generate-manifest.sh

```diff
@@ -27,7 +27,7 @@ Generate a YAML manifest for Antrea using Kustomize and print it to stdout.
         --kind                Generate a manifest appropriate for running Antrea in a Kind cluster
         --cloud               Generate a manifest appropriate for running Antrea in Public Cloud
         --ipsec               Generate a manifest with IPSec encryption of tunnel traffic enabled
-        --np                  Generate a manifest with Namespaced Antrea NetworkPolicy CRDs enabled
+        --np                  Generate a manifest with Namespaced Antrea NetworkPolicy CRDs and NetworkPolicy related CRDs enabled
```

Comment:
nit: maybe just `Generate a manifest with support for Antrea Namespaced NetworkPolicy CRDs and Antrea Cluster NetworkPolicy CRDs`

I don't think the concept of "NetworkPolicy related CRDs" need to come up here